src/parser/github-comment-module.ts

```diff
@@ -222,13 +222,13 @@ export class GithubCommentModule implements Module {
             <td>
             <details>
               <summary>
-                ${new Decimal(commentScore.score?.words?.result || 0).add(new Decimal(commentScore.score?.formatting?.result || 0))}
+                ${new Decimal(commentScore.score?.words?.result ?? 0).add(new Decimal(commentScore.score?.formatting?.result ?? 0))}
               </summary>
               <pre>${formatting}</pre>
              </details>
             </td>
-            <td>${commentScore.score?.relevance || "-"}</td>
-            <td>${commentScore.score?.reward || "-"}</td>
+            <td>${commentScore.score?.relevance ?? "-"}</td>
```

Comment:
It seems like you're going this everywhere. I thought it's the opposite of || but I'm probably mistaken. Can you explain why you're changing these?

Reply:
> The ?? operator only checks for null or undefined values, while the || operator also checks for falsy values such as 0, false, NaN, or an empty string. In some cases, using || can unintentionally filter out valid values like 0. Therefore, replacing || with ?? is usually done to ensure that valid falsy values (like 0) are preserved.

Learned something new.